intf/intf.go

```diff
@@ -35,12 +36,17 @@ type Interface struct {
 	MAC net.HardwareAddr
 }
 
+// String is the string representation of an interface that can be read by humans.
+func (i Interface) String() string {
+	return fmt.Sprintf("%s (index %d, MAC: %s)", i.Name, i.Index, i.MAC)
+}
+
 // ARPEntry is a representation of an ARP entry on the underlying system.
 type ARPEntry struct {
 	// IP is the IP address of the neighbour.
 	IP net.IP
 	// Interface is the name of the interface on which the ARP entry was learnt.
-	Interface string
+	Interface *Interface
```

Comment:
github doesn't let me comment on other parts on the code...this comment is meant for line 70. The comment there is unfinished?

Reply:
Thanks - finished this sentence (just needed a period).